src/log-component/providers/log-provider.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// tslint:disable:no-any
+import {Provider} from '@loopback/context';
+import {ParsedRequest} from '@loopback/core';
+
+export interface LogFn {
+  (req: ParsedRequest, args: any, result: any, startTime?: number[]): void;
```

Comment:
can `args` and/or `result` be strictly typed? result probably won't, but I'm wondering if `args` can be.

Reply:
`args` can't be typed because the user specifies them for each route so they can be different. We have a `OperationArgs` type which is of type `any[]` so I'm not sure changing it to that is going to make a difference.

Reply:
Yeah then leaving it as `any` is fine by me.